src/_includes/templates/nav/about-us-media.html

```diff
@@ -1,9 +1,9 @@
 <a href="/the-bureau/">
     <img src="/static/img/nav-about-us-video.png"
          alt="About Us video image" />
-    <h4 class="short-desc u-mt15">
+    <div class="h4 short-desc u-mt15">
```

Comment:
I think I made it a div to keep it display block like a `h` element, but I think that might be included in the `.h4` class. I'm going to see if I can change these all to `span`s. Good call.